runtime/syntax/vue.yaml

```diff
@@ -56,9 +56,28 @@ rules:
         rules:
             - include: "typescript"
 
+    - default:
+        start: "<script[ ]+setup>"
+        end: "</script>"
+        rules:
+            - include: "javascript"
```

Comment:
I suggest to merge/combine this rule with L46..51, since it will reduce the amount of rules to check. Let Regexp take care of different expressions build with ORs and occurrence checks.

BTW:
Is the `limit-group: symbol.tag` in L49 really necessary? Because it will limit the rules inside the included `javascript` to `symbol.tag` only.